src/Data/Nat/DivMod.agda

```diff
@@ -255,6 +294,78 @@ m*n/m*o≡n/o m@(suc m-1) n o = helper (<-wellFounded n)
     n / o                         ∎
     where n∸o<n = ∸-monoʳ-< (n≢0⇒n>0 (≢-nonZero⁻¹ o)) (≮⇒≥ n≮o)
 
+m<n*o⇒m/o<n : ∀ {m n o} → m < n * o → .⦃ _ : NonZero o ⦄ → m / o < n
+m<n*o⇒m/o<n {m} {suc n} {o} m<n*o with m <? o
+... | yes m<o = begin-strict
+  m / o ≡⟨ m<n⇒m/n≡0 m<o ⟩
+  0     <⟨ s≤s z≤n ⟩
+  suc n ∎
+... | no ¬m<o = begin-strict
```

Comment:
Very minor but can we use the unicode `≮`?

Reply:
Right. That would be consistent with things like `≮⇒≥`.